woodwork/utils.py

```diff
@@ -274,10 +274,14 @@ def _is_valid_latlong_series(series):
 
 def _is_valid_latlong_value(val):
     '''Returns True if the value provided is a properly formatted LatLong value, otherwise
-    returns False'''
+    returns False.'''
     if isinstance(val, tuple) and len(val) == 2:
         if isinstance(val[0], float) and isinstance(val[1], float):
+            if pd.isnull(val[0]) and pd.isnull(val[1]):
```

Comment:
bit of a nitpick, but it might read nicely to have `latitude, longitude = val` and then reference those variables in this line and the once above.

Reply:
Updated